docs/intermediate/tutorial11-normals/README.md

```diff
@@ -151,12 +151,13 @@ Some model formats include the tanget and bitangent (sometimes called the binorm
 
 ![](./tangent_space.png)
 
-Basically we can use the edges of our triangles, and our normal to calculate the tangent and bitangent. But first, we need to update our `ModelVertex` struct in `model.rs`.
+Basically we can use the edges of our triangles, and our normal to calculate the tangent and bitangent. But first, we need to update our `ModelVertex` struct in `model.rs`. In addition to adding two new fields `tangent` and `bitangent`, we also need to change the types from `[f32, N]` to `cgmath::VectorN<f32>` so that we can do some matrix calculations later.
```

Comment:
So the reason why I start using `cgmath` here is because I need to do some arithmetic to calculate the bitangents, and tangents, not to do matrix calculations. You can't add and subtract `[f32; 3]`. I could write the required traits to do so, but `cgmath` does that for me.

Reply:
It would probably be better to include this info as a comment on `ModelVertex`.

Reply:
Sounds good, thanks!